go.mod

```diff
@@ -44,6 +44,7 @@ require (
 	k8s.io/utils v0.0.0-20250604170112-4c0f3b243397
 	sigs.k8s.io/controller-runtime v0.21.0
 	sigs.k8s.io/controller-tools v0.18.0
+	sigs.k8s.io/crdify v0.4.0
```

Comment:
Note: I believe that this particular import version removes the behavior added in #2023 

https://github.com/kubernetes-sigs/crdify/pull/16 merged recently so pulling the latest commit in should fix this